assets/styles/layout/_options.scss

```diff
@@ -28,15 +28,6 @@
     }
   }
 
- &__layout{
```

Comment:
it seemed like this was trying to do something to keep the border radius only effecting the right most option the layout before was

- `menu`
  - `li`
- `menu`
  - `li`
  - `li`

I changed this to just be one `menu` with 3 `li`s and made the css selector simpler with `:not last`, `:last` so it's always accurate no matter how many items exist